.github/actions/check_links/index.js

```diff
@@ -38,9 +38,10 @@ async function run() {
 
   try {
     // Run the link check against the PR preview link
-    const output = execSync(
-      `../../../website/node_modules/.bin/linkcheck ${deployUrl}`
-    );
+    // const output = execSync(
+    //   `../../../website/node_modules/.bin/linkcheck ${deployUrl}`
+    // );
+    const output = execSync(`ls ../../../website/node_modules/.bin`);
```

Comment:
for path related stuff i'd check out `GITHUB_WORKSPACE`.

Try logging `process.env.GITHUB_WORKSPACE` and working from there.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables